modules/caddyhttp/reverseproxy/healthchecks.go

```diff
@@ -153,6 +156,12 @@ func (a *ActiveHealthChecks) Provision(ctx caddy.Context, h *Handler) error {
 	a.httpClient = &http.Client{
 		Timeout:   timeout,
 		Transport: h.Transport,
+		CheckRedirect: func(req *http.Request, via []*http.Request) error {
+			if !a.FollowRedirects {
+			    return http.ErrUseLastResponse
```

Comment:
@mohammed90 This lint error is a bit confusing, since this is a `gofmt` error it looks like spaces instead of tabs.

@aliasgar55 Would you be able to `go fmt` this file?

Reply:
That was quick. haha

Reply:
I think it's because golangci-lint runs the linters in the order we specify them in `.golangci.yml` file, and `gci` comes before `gofmt` there.

Reply:
Hmm, but I guess why does gci care about go fmt output? I thought it was about imports.